lib/std/crypto/utils.zig

```diff
@@ -138,6 +138,62 @@ pub inline fn secureZero(comptime T: type, s: []T) void {
     @memset(@as([]volatile T, s), 0);
 }
 
+const valgrind_support = @import("builtin").valgrind_support;
+
+fn markSecret(ptr: anytype, comptime action: enum { classify, declassify }) void {
+    if (@inComptime()) return;
```

Comment:
```suggestion
    if (!std.debug.inValgrind()) return;
```

[inValgrind](https://ziglang.org/documentation/master/std/#std.debug.inValgrind)

Also I think this check should be pushed up into `classify` and `declassify`.